powershell-analyzer/action.yml

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
Can we also get an action in this repository that demonstrates use of `analyze.ps1` ... on this repository itself?